eng/Directory.Build.Common.props

```diff
@@ -20,16 +20,28 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
+    <ContinuousIntegrationBuild>false</ContinuousIntegrationBuild>
+    <ContinuousIntegrationBuild Condition="'$(TF_BUILD)' == 'true'">true</ContinuousIntegrationBuild>
+    <CI>$(ContinuousIntegrationBuild)</CI>
```

Comment:
Why do we need both `CI` and `ContinuousIntegrationBuild`?

Reply:
Let me check!